docs/guides/features/system-tray.md

```diff
@@ -193,10 +193,15 @@ Note that you need to add `icon-ico` or `icon-png` feature flag to the tauri dep
 app.tray_handle().set_icon(tauri::Icon::Raw(include_bytes!("../path/to/myicon.ico").to_vec())).unwrap();
 ```
 
-### Keep the app running in the background after closing all windows
+### Preventing the App from Closing
 
-By default, tauri closes the application when the last window is closed.
-If your app should run in the background, you can call `api.prevent_close()` like so:
+By default, tauri closes the application when the last window is closed. We can simply call `api.prevent_close()` to prevent this.
+
+Depending of your need you can use one of the two following exemples:
+
+#### Keeping the Backend Running in the Background
```

Comment:
```suggestion
**Keep the Backend Running in the Background**
```

Reply:
Changing to a bold instead of heading should hopefully help a bit (but admittedly I didn't preview it live)